integrationTests/vm/delegation/delegation_test.go

```diff
@@ -77,10 +79,13 @@ func TestDelegationSystemSCWithValidatorStatistics(t *testing.T) {
 
 	round, nonce = processBlocks(t, round, nonce, 1, nodesMap)
 
-	for _, node := range nodes {
-		txData := "changeRewardAddress" + "@" + hex.EncodeToString(rewardAddress)
-		integrationTests.CreateAndSendTransaction(node, nodes, big.NewInt(0), vm.ValidatorSCAddress, txData, core.MinMetaTxExtraGasCost)
-		delegateToSystemSC(node, nodes, rewardAddress, big.NewInt(1000000))
+	for index, node := range nodes {
+		if index == nodeIndexForDelegationOwner {
```

Comment:
This is a strange resolve.

Reply:
As explained in the short meeting: the test was made in such a way that all nodes changed the rewards to the delegation SC. Now, since that "feature" no longer exists, I choose to do merges. Since the owner can not be whitelisted, I put an exception for it.